### PR TITLE
Add content for the wiki

### PR DESCRIPTION
**Pull request by nahteb**
This adds more content to the wiki for our own rulesets.

### CODE REVIEW

File: config/locales/rules.yml [hunk 1]
```diff
@@ -9,8 +9,35 @@ en:
...
+
+      The version of the OpenAPI document must be version 3.1.0. OpenAPI v3.1.0 is the most recent official version.
+      It contains more features than the previous versions. You can read more about the differences in a [blog post on the Stoplight site](https://blog.stoplight.io/difference-between-open-v2-v3-v31).
+
+      Example
+      -------
+
+      The following is an example of how your OpenAPI document that is using an old version:
+
+
```
Comment (RobNicholsGDS) [inline config/locales/rules.yml:25]: [pedantic]
Remove second empty line (double empty line)

File: config/locales/rules.yml [hunk 2]
```diff
@@ -9,8 +9,35 @@ en:
...
+      The following is an example of how your OpenAPI document that is using an old version:
+
+  
+          openapi: 3.0.0
+          info:
+            title: ''
+            version: '1.0.0'
+
+
```
Comment (RobNicholsGDS) [inline config/locales/rules.yml:31]: [pedantic]
Remove second empty line (double empty line)

File: config/locales/rules.yml [hunk 3]
```diff
@@ -24,7 +51,7 @@ en:
       Example
       -------
 
-      The following is an example of an OpenAPI document which is referencing a closed source license type:
+      The following is an example of an OpenAPI document which is referencing a proprietary license type:
```
Comment (RobNicholsGDS) [inline config/locales/rules.yml:56]: [pedantic]
Remove second empty line (double empty line)

**Review by RobNicholsGDS — COMMENTED** [review]
Looks good - but needs testing locally.

There are a couple of double empty lines that it would be nice to remove (one of them (65-66) is mine I'm afraid 🤭 )

File: config/locales/rules.yml (hunk 1 above)
Comment (nahteb) [reply config/locales/rules.yml]: done

File: config/locales/rules.yml (hunk 2 above)
Comment (nahteb) [reply config/locales/rules.yml]: done

File: config/locales/rules.yml (hunk 3 above)
Comment (nahteb) [reply config/locales/rules.yml]: done

**Review by RobNicholsGDS — APPROVED** [review]
LGTM

**Pull request merged**